src/chains/ethereum/options/src/wallet-options.ts

```diff
@@ -274,15 +284,22 @@ export const WalletOptions: Definitions<WalletConfig> = {
     cliAliases: ["u", "unlock"],
     cliType: "array:string"
   },
-  secure: {
+  lock: {
     normalize,
     cliDescription:
       "Lock available accounts by default (good for third party transaction signing).",
     default: () => false,
-    legacyName: "secure",
-    cliAliases: ["n", "secure"],
+    legacyName: "lock",
+    cliAliases: ["n", "secure", "lock", "lock"],
```

Comment:
```suggestion
    cliAliases: ["n", "secure", "lock"],
```

Reply:
you sure you don't mean
```suggestion
    cliAliases: ["n", "secure", "lock", "lock", "lock"],
```